docs/tutorial/t-spark-streaming.md

```diff
@@ -0,0 +1,335 @@
+## Streaming workloads with Charmed Spark
```

Comment:
This is a really long page that is easy to get lost in - it would be helpful to break it down into sections and list them at the beginning to prepare the user.

Here are the sections I identified in the story:
- Set up new workspace
- Set up Kafka and Zookeper 
- Generate sample streaming data 
- Get credentials for Kafka
- Write a Spark job in Python
- View results